test/cli/src/test_functions.py

```diff
@@ -831,7 +831,7 @@ def test_functions_install_provisioner_rpm(
         'cat /etc/yum.repos.d/prvsnr.repo | grep baseurl'
     ).split('=')[1]
     assert baseurl == (
-        'http://cortx-storage.colo.seagate.com/releases/cortx/{}'
+        'http://<cortx_release_server>/releases/cortx/{}'
```

Comment:
@andkononykhin2 , @dmitrydb 
Need your attention here. What could be the best alternative?

Reply:
The same:  please just add FIXME note. We might not need that test anymore actually.